src/HealthKit/EnumsCompat.cs

```diff
@@ -15,6 +15,8 @@ public unsafe static partial class HKQuantityTypeIdentifierKey  {
 			}
 		}
 
+		[Introduced (PlatformName.iOS, 9,3, PlatformArchitecture.All)]
+		[Introduced (PlatformName.WatchOS, 2,2, PlatformArchitecture.All)]
```

Comment:
it's already on the `getter` ?
looks like the test code is wrong (and should be fixed)
but we should not duplicate it on both the property and the getter (too much metadata if the linker is not enabled)

Reply:
The generator puts the attributes on both the getter and the property:

```csharp
[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
[Introduced (PlatformName.iOS, 9,0, PlatformArchitecture.All)]
[Introduced (PlatformName.MacOSX, 10,11, PlatformArchitecture.All)]
public static NSString ColorPrimaries_DCI_P3 {
	[Introduced (PlatformName.iOS, 9,0, PlatformArchitecture.All)]
	[Introduced (PlatformName.MacOSX, 10,11, PlatformArchitecture.All)]
	get {
		return CVImageBufferColorPrimaries.DciP3.GetConstant () as NSString;
	}
}
```

I agree that it should be redundant to put the attributes on both, but I think that's best fixed in a different PR.

Reply:
ok, there must be a reason (what the IDE display) for the duplication. Better find it (again) before fixing it :)